### DISCUSSION TITLE
Processing a file and change sections based on config

### DISCUSSION THREAD

**Discussion by ozanbaba** [mesonbuild/meson#10232] [Q&A] [answered]
Hello All,

I am working on moving evolution-on to meson to learn more about meson. Now I so far got the compiling, installation of plugin are done. But I need to change sections of eplug file so it would point to the installation folder, gettext package name, and localedir. I checked the documentation and didn't really come by anything  for it.  What am I missing here?

Regards,
Ozan

**Comment by mensinda** [comment, marked as answer]
Did you already have a look at [`configure_file`](https://mesonbuild.com/Reference-manual_functions.html#configure_file)?

**Comment by ozanbaba** [comment]
That worked perfectly now.  Thank you.
